server/helpers/forgotPasswordEmail.js

```diff
@@ -0,0 +1,9 @@
+const resetPasswordEmail = resetUrl => ({
+  subject: 'Authors Haven Password Reset',
+  body:
+    `<p>
+      Click <a href="${resetUrl}">here</a> to reset your password.
+    </p>`
+});
+
+export default resetPasswordEmail;
```

Comment:
A base filename should exactly match the name of its default export. Your file name is _forgotPasswordEmail.js_ but your default export is _resetPasswordEmail_

Reply:
My mistake, I will sort this out.